examples/templates/vue/README.md

```diff
@@ -0,0 +1,61 @@
+# {{ YOUR TITLE GOES HERE }}
+
+## Description
+
+{{ YOUR DESCRIPTION GOES HERE }}
+
+## Live preview
+
+You can preview this template on CodeSandbox  under the URL: https://githubbox.com/handsontable/code-snippets/tree/master/examples/templates/vue
+
+## How to run this project
+
+### Installation
+
+Call `npm install` to install all necessary dependencies. This command will first remove `node_modules` folder and install testing tools (`jasmine`, `jasmine-console-reporter`, `puppeteer` and `http-server`) from the root directory.
+
+### Development
+
+To start local development server call `npm run start`. Now you can visit http://localhost:8080 to view this project.
+
+### Testing
+
+In order to run tests for this project you have two options:
+
+1. Call `npm run start` which will launch the development server and after the server is running, call `npm run jasmine` to run test specs.
+2. Call `npm run test` which will build the project, then start the server on the production output folder, run test specs and close the server.
+
+## License
+
+Handsontable is dual-licensed. You can either use a free license for all your non-commercial projects or purchase a commercial license. See the table below for a comparison of these two.
+
+<table>
+  <thead align="center">
+    <tr>
+      <th width="50%">Free license</th>
+      <th width="50%">Paid license</th>
+    </tr>    
+  </thead>
+  <tbody align="center">
+    <tr>
+      <td>For non-commercial purposes such as teaching, academic research, personal experimentation, and evaluating  on development and testing servers.</td>
+      <td>For all commercial purposes</td>
+    </tr>
+    <tr>
+      <td>All features are available</td>
+      <td>All features are available</td>
+    </tr>
+    <tr>
+      <td>Community support</td>
+      <td>Dedicated support</td>
+    </tr>    
+    <tr>
+      <td><a href="//github.com/handsontable/handsontable/blob/master/LICENSE.txt">Read the license</a></td>
+      <td><a href="//handsontable.com/pricing">See plans</a></td>
+    </tr>
+  </tbody>
+</table>
+
+## Contact support
+
+We provide support for all users through [GitHub issues](https://github.com/handsontable/vue-handsontable-official/issues). If you have a commercial license then you can add a new ticket through the [contact form](https://handsontable.com/contact?category=technical_support).
```

Comment:
Please check all the links. The github.com/handsontable/vue-handsontable-official will no longer be valid after Monorepo. All links should link to our handsontable/handsontable repo

Reply:
Done https://github.com/handsontable/handsontable/pull/7515/commits/4cedf4a98e61e6ca3bd794aacadbac53b0f563fa